Sources/SwiftRefactor/CallToTrailingClosures.swift

```diff
@@ -0,0 +1,131 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftBasicFormat
+import SwiftSyntax
+
+/// Convert a call with inline closures to one that uses trailing closure
+/// syntax. Returns `nil` if there's already trailing closures or there are no
+/// closures within the call. Pass `conversionStart` to specify an index to
+/// start the conversion from, ie. to skip converting closures before
+/// `conversionStart`
+///
+/// ## Before
+/// ```
+/// someCall(closure1: { arg in
+///   return 1
+/// }, closure2: { arg in
+///   return 2
+/// })
+/// ```
+///
+/// ## After
+/// ```
+/// someCall { arg in
+///   return 1
+/// } closure2: { arg in
+///   return 2
+/// }
+/// ```
+public struct CallToTrailingClosures: SyntaxRefactoringProvider {
+  public struct Options {
```

Comment:
You could just name this struct `Context` and then you wouldn’t need the `typealias` below.